homeassistant/components/route53/__init__.py

```diff
@@ -84,16 +84,12 @@ def _update_route53(
 
     # Get the IP Address and build an array of changes
     try:
-        ipaddress = get_ip()
+        ipaddress = requests.get("https://api.ipify.org/", timeout=5).text()
```

Comment:
We could make this a function in our util package.

Reply:
Let's do that outside of this hot fix.

Reply:
`text` is a string, and not callable.
PR to correct that <https://github.com/home-assistant/core/pull/38100>